### PR TITLE
add markup and styles

### PR DESCRIPTION
**Pull request by nataliematiushenko**
This is a responsive web page.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+}
+
+/* Positioning inside divs */
+
+.header-div,
+#logo-section,
+#div-three-one,
+#div-three-two,
+.fourth,
+#section5,
```
Comment (karmeljuk) [inline style.css]: it's better don't use id in css at all

File: style.css [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+    height: auto;
+  }
+
+}
+
+@media (min-width: 1024px) {
+
+/* ==========LOGO-SECTION========== */
+
+  #logo-section {
```
Comment (karmeljuk) [inline style.css]: change id to class

**Review by karmeljuk — CHANGES_REQUESTED** [review]
minor changes: id to class

**Comment by nataliematiushenko** [comment]
Sorry for messing up with pull requests and commits.
Have hanged selectors in both files.

**Review by karmeljuk — COMMENTED** [review]
code ok

**Pull request merged**